packages/common-utils/src/metadata.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { renderChartConfig } from '@/renderChartConfig';
 import type { ChartConfig, ChartConfigWithDateRange, TSource } from '@/types';
 
-export const DEFAULT_MAX_ROWS_TO_READ = 1e6;
+export const DEFAULT_MAX_ROWS_TO_READ = 3e6;
```

Comment:
can you add a comment as to why this number was chosen? if it's arbitrary, it's fine. but it might be nice to know when we might consider adjusting it, etc...